lib/bitbucket_rest_api/response/raise_error.rb

```diff
@@ -11,6 +13,9 @@ def on_complete(env)
       when 400
         raise BitBucket::Error::BadRequest.new(env)
       when 401
+        if env[:body] =~ REFRESH_TOKEN
```

Comment:
I guess there's no way to determine if its a 'retriable' 401 without using regex to parse the message?

Reply:
Not that I can think of and unfortunately, I have not been able to reproduce this locally so I am sort of flying blind.